ts-orm/README.md

```diff
@@ -0,0 +1,51 @@
+# ORM Sample App
+
+## Run the app locally
+
+## Compile and Deploy with Cloudcc
+
+run the terminal commands:
+```sh
+# Compile the app
+tsc && klotho ./dist/index.js --app sample-ts-orm
```

Comment:
might be worth splitting out the tsc call for consistency w/ the other samples and just for clarity in case folks are unsure of how/what tsc is

Reply:
sure, i can do that